.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,37 @@
+name: integration-test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  integration-test:
+    name: Run Integration- Test
+    runs-on: ubuntu-latest
+    environment: Integration Test
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
We could use the github actions OIDC provider instead, so we don't have to remember to rotate the creds.

See https://github.com/aws-actions/configure-aws-credentials#assuming-a-role and https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services

Reply:
Thanks, we should do that.
It was my original intent, but I hit some policy snags setting up our testing account. Let me take another crack at it for the next revision.